config/configureEnv.js

```diff
@@ -40,8 +40,9 @@ const configureEmail = async () => {
       return email;
     }
     const answer = await askFor("email");
+    email = answer.email;
```

Comment:
```js
const { answer } = email;
```
hence 
```js
const { email } = await askFor...
```

you wouldn't be doing email.email, the entire point of destructuring would be to not have to reach into an object at all - i.e. the following line would become `return email;`

Reply:
ahhhhhhhh makes sense